Tests/Parser/Client/fixtures/browser.yml

```diff
@@ -2012,8 +2012,8 @@
     type: browser
     name: Qwant Mobile
     version: "1.3"
-    engine: Gecko
-    engine_version: ""
+    engine: WebKit
+    engine_version: "604.1.38"
     family: Firefox
```

Comment:
The browser family is still being marked as Firefox here (since all the Qwant Mobile entries use the same browser and that browser is defined as part of the Firefox family). Should I change that? If so, what should I change it to?

Reply:
for iOS, everything that is a browser is always a Webkit engine, more precisely safari. Therefore, it does not matter which browser you put on your phone, it will be safari and have similar +- inconveniences for developers and the same display errors.

so the current changes for iOS are correct